docs/examples/hello_world/README.md

```diff
@@ -69,9 +69,9 @@ parameters:
 
 <2> Provisioner
 
-<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints)
+<3> GlusterFS defined EndPoint taken from Step 1 above (kubectl get endpoints). As of Kubernetes 1.7, this parameter should be removed as Kubernetes will reject this YAML definition.
```

Comment:
Oh, right, yes. Should we just remove the endpoint parameter form the demo in general?